qri/dataset.py

```diff
@@ -34,7 +34,7 @@ def render(self):
         return markdown.markdown(self.script)
 
     def __repr__(self):
-        return self.script
+        return self.script or ''
```

Comment:
@dustmop I saw you put the handle comment in the `__init__` class but my guess was it's best to handle it here since we're returning the obj representation when asking for a dataset's readme. Am I off track? I also saw in your issue you intended to return `None` but I thought this method needed to return a string so I just made it empty. Open to feedback on how that can be more declarative though

Reply:
Fixing it here looks like the right thing to do, I just wasn't sure where it should go.

However, thinking it over a bit more, I feel it's better to return the actual string `'None'` if the readme doesn't exist. The `__repr__` *must* return a string, but the string should be a representation of the data, not just a stringified version. For example, `__repr__` for the string `"hello"` will contain the outer quotes. Another way to think about this is that the result of `__repr__` should be parsable to get back the original value, which is sometimes stated as `eval(repr(c)) == c`.

We're playing a bit fast and loose with that here, as the readme's repr is just returning its text contents. It should probably have the same type wrapper that Meta and Structure are using.

In summary, I think this is fine for fixing the problem at hand! I'll follow up with another commit to handle the related problems mentioned.